fmskill/model/pandas.py

```diff
@@ -92,6 +92,8 @@ def item_name(self):
         return self.df.columns[0]
 
     def __init__(self, df, name: str = None, item=None):
+        self.itemInfo = eum.ItemInfo(eum.EUMType.Undefined)
```

Comment:
How about passing ItemInfo as an argument  to  `__init__`, instead of setting it afterwards. Default behavior can still be EUMType.Undefined?

Reply:
Great idea - let's do that!